SMLHelper/Handler.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace SMLHelper.V2
+{
+    using Interfaces;
+
+    /// <summary>
+    /// A simple location where every SMLHelper handler class can be accessed.
+    /// </summary>
+    public static class Handler
```

Comment:
Any recommendations on a better name for this?

Reply:
`Handlers`

Reply:
That name isn't possible because that is already the name of the namespace.

Reply:
Oh, I didn't see the namespace.

In that case, why is it in the root folder and not in the Handlers folder?

Reply:
In trying to make this class itself as easy to discover as possible

Reply:
So we're getting rid of static methods?

Reply:
I wouldn't get rid of the static methods unless we are willing to completely break backwards compatibility.  
This here is just to make it easier for modders to find all the handlers in one place without needing to look up the documentation.